gradle-versions-plugin/build.gradle

```diff
@@ -41,14 +50,16 @@ dependencies {
   testImplementation 'org.spockframework:spock-junit4:2.1-M2-groovy-3.0', { exclude module: 'groovy-all' } // Use localGroovy()
 }
 
-task docsJar(type: Jar, dependsOn: dokkaHtml) {
+tasks.register('docsJar', Jar) {
+  dependsOn dokkaHtml
```

Comment:
and here I think can be done via `java.withJavadocJar()`

Reply:
I think we have this custom task for dokka.